test/plugins/elasticsearch.spec.js

```diff
@@ -56,15 +56,28 @@ describe('Plugin', () => {
           client.search({ index: 'logstash-2000.01.01' }, () => {})
         })
 
+        it('should sanitize the path name', done => {
+          agent
+            .use(traces => {
+              expect(traces[0][0].meta).to.have.property('elasticsearch.url', '/logstash-?.?.?/_search')
+            })
+            .then(done)
+            .catch(done)
+
+          client.search({ index: 'logstash-2000.01.01' }, () => {})
```

Comment:
do you mind creating a document by id w/ an op_type query parameter instead?

Reply:
Updated.